src/ml_modules/base_classifier.py

```diff
@@ -0,0 +1,9 @@
+from abc import ABC, abstractmethod
+from typing import List
+
+
+class BaseClassifier(ABC):
+
+    @abstractmethod
+    def classify(self, text: str) -> List[str]:
+        raise NotImplementedError
```

Comment:
I know we're using off repo eval, but for local eval we also need a "all_intents" method that the classifier should use to advertise which intents it can classify.